build.sbt

```diff
@@ -8,7 +8,7 @@ ThisBuild / version      := sys.env.getOrElse("BUILD_VERSION", "dev-SNAPSHOT")
 
 
 val cpgVersion        = "1.4.22"
-val joernVersion      = "2.0.92"
```

Comment:
Don't upgrade to 98 we are seeing delta in comparison report

Reply:
do you know which joern version is stable at current state? anyways we need to update to latest as Go's fixes are on the way.

Reply:
we can upgrade till 2.0.92, I guess there is already a PR created by Datta on Dev